### DISCUSSION TITLE
How to add a google-site-verification meta-tag

### DISCUSSION THREAD

**Discussion by franckl** [squidfunk/mkdocs-material#7809] [Q&A]
Is there a way to add a custom meta tag ? The Meta-Data extension doesn't seem to help.

I need to verify the ownership of a subdomain using the special google tag:
`<meta name="google-site-verification" content="XXXX" />`

**Comment by squidfunk** [comment]
Habe you read the docs on customization?
https://squidfunk.github.io/mkdocs-material/customization/#overriding-template-blocks

The `extrahead` block should do the trick.

**Comment by squidfunk** [comment]
Closing as answered.

**Comment by do-me** [comment]
Just to have the complete answer here:
In your `overrides/partials` folder create `main.html` 
```
.
├─ overrides/
│  └─ main.html
└─ mkdocs.yml
```

with the following content

```html
{% extends "base.html" %}

{% block extrahead %}
  <meta name="google-site-verification" content="your-key" />
  {{ super() }}
{% endblock %}
```

The file mustn't be called `main.html`, it can have any name. It even worked with `base.html` but this should probably be avoided.

**Comment by squidfunk** [comment]
Thanks for sharing! ❤️

**Comment by VityaSchel** [comment]
> Just to have the complete answer here: In your `overrides/partials` folder create `main.html`
> 
> ```
> .
> ├─ overrides/
> │  └─ main.html
> └─ mkdocs.yml
> ```
> 
> with the following content
> 
> ```
> {% extends "base.html" %}
> 
> {% block extrahead %}
>   <meta name="google-site-verification" content="your-key" />
>   {{ super() }}
> {% endblock %}
> ```
> 
> The file mustn't be called `main.html`, it can have any name. It even worked with `base.html` but this should probably be avoided.

This does not work for me, I tried main.html and base.html yet this meta tag does not appear neither in mkdocs serve nor built directory. I'm using material theme

**Comment by rajtilakjee** [comment]
@VityaSchel Here's what I did:

1. Created a folder called `overrides` at the same level with that of mkdocs.yml
2. Added a file called `main.html` in this folder
3. Added the following code in this file:
```
{% extends "base.html" %}

{% block extrahead %}
  <meta name="google-site-verification" content="--code here--" />
{% endblock %}
```

That's it, give it some time (5-10 minutes) before you verify your site with Google. You can see my site for example: https://github.com/rajtilakjee/parseltongue
